pyiceberg/catalog/__init__.py

```diff
@@ -661,6 +661,14 @@ def create_table_transaction(
         )
 
     def table_exists(self, identifier: Union[str, Identifier]) -> bool:
+        """Check if a table exists.
+
+        Args:
+            identifier (str | Identifier): Table identifier.
+
+        Returns:
+            bool: True if the table exists, False otherwise.
+        """
```

Comment:
I think it can automatically inherit the docstring from the super class' `Catalog.table_exists`. Shall we remove these such that if anything need to be changed in the future, we only have to update the docstring in class `Catalog`?

Reply:
oh is that, I was unsure about that, if so then we can definitely go ahead and remove!

Reply:
Seems it is not removed in the last commit. Could you please remove it?